File: Corredor/Digitales/Go/escuela.go
package main

import "fmt"

type Escuela struct {
	nombre      string
	direccion   string
	profesores  []Profesor
	estudiantes []Estudiante
}

func NuevaEscuela(nombre, direccion string) Escuela {
	return Escuela{
		nombre:      nombre,
		direccion:   direccion,
		profesores:  []Profesor{},
		estudiantes: []Estudiante{},
	}
}

func (e *Escuela) AgregarProfesor(p Profesor) {
	e.profesores = append(e.profesores, p)
}

func (e *Escuela) AgregarEstudiante(est Estudiante) {
	e.estudiantes = append(e.estudiantes, est)
}

func (e Escuela) MostrarInfo() {
	fmt.Printf("🎵 Escuela: %s | Dirección: %s\n", e.nombre, e.direccion)
	fmt.Printf("Profesores: %d | Estudiantes: %d\n", len(e.profesores), len(e.estudiantes))
}

File: Corredor/Digitales/C#/C#/Escuela.cs
using System;
using System.Collections.Generic;

namespace EscuelaMusica
{
    public class Escuela
    {
        public string Nombre { get; set; }
        public string Direccion { get; set; }
        public List<Profesor> Profesores { get; set; }
        public List<Estudiante> Estudiantes { get; set; }

        public Escuela(string nombre, string direccion)
        {
            Nombre = nombre;
            Direccion = direccion;
            Profesores = new List<Profesor>();
            Estudiantes = new List<Estudiante>();
        }

        public void AgregarProfesor(Profesor p)
        {
            Profesores.Add(p);
        }

        public void AgregarEstudiante(Estudiante e)
        {
            Estudiantes.Add(e);
        }

        public void MostrarInfo()
        {
            Console.WriteLine($"🎵 Escuela: {Nombre}, Ubicación: {Direccion}");
            Console.WriteLine($"Profesores: {Profesores.Count}, Estudiantes: {Estudiantes.Count}");
        }
    }
}